CMakeLists.txt

```diff
@@ -20,15 +20,13 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
 option(MANIFOLD_EXPORT off)
 option(MANIFOLD_DEBUG off)
-option(MANIFOLD_USE_CUDA on)
-set(MANIFOLD_PAR "TBB" CACHE STRING "Parallel backend, either \"TBB\" or \"OpenMP\" or \"NONE\"")
```

Comment:
I changed these defaults for minimal dependency. @pca006132 Does that look reasonable to you?